src/main/scala/temple/generate/database/PostgresGenerator.scala

```diff
@@ -61,8 +72,16 @@ object PostgresGenerator extends DatabaseGenerator {
             .mkString(",\n")
             .pipe(indent(_))
         s"CREATE TABLE $tableName (\n$stringColumns\n);"
-      case Read(tableName, columns) =>
+      case Read(tableName, columns, condition) =>
+        val stringColumns = columns.map(_.name).mkString(", ")
+        val stringCondition = condition.map(generateCondition) match {
+          case Some(conditionString) => List(s"WHERE $conditionString")
```

Comment:
Why is this a list? Surely we just want to produce the empty string if none?

Reply:
I can see the reasoning behind this, as it determines the spacing before. The alternative would be to stick a space before the word WHERE, not sure if that makes things clearer